rfi_file_monitor/utils/__init__.py

```diff
@@ -52,6 +52,12 @@ def get_patterns_from_string(input: str) -> List[str]:
     # multiple patterns are supported, provided they are separated with commas
     return list(map(lambda x: x.strip(), input.split(',')))
 
+def get_ignore_patterns_from_string(input: str, defaults=[]) -> List[str]:
```

Comment:
Is it possible to combine `get_patterns_from_string` and `get_ignore_patterns_from_string` into a single method?